helm/configuration/routes.json

```diff
@@ -184,5 +196,29 @@
     "method": "DELETE",
     "handler": "revertDiscoveryData",
     "on_microservice": "openehr_service"
+  },
+  {
+    "uri": "/mpi/Patient/:id",
```

Comment:
keep URL lowercase -> `"/mpi/patient/:id"`

Reply:
BTW, I think these handlers should be removed because there is no `mpi_service` in the project and it missed in this PR

Reply:
P.S.
createPatient -> postPatient
editPatient -> putPatient